tests/core/coredump-options/task.yaml

```diff
@@ -6,16 +6,33 @@ details: |
 
 systems: [-ubuntu-core-1*, -ubuntu-core-20*, -ubuntu-core-22*]
 
+restore: |
+  rm -f /var/lib/systemd/coredump/*
+
 execute: |
+  # To get VERSION_ID defined
+  . /etc/os-release
+
   cfg_path=/etc/systemd/coredump.conf.d/ubuntu-core.conf
 
   # coredumps should be initially disabled
   expect=$(printf "[Coredump]\nStorage=none\nProcessSizeMax=0\n")
   test "$expect" = "$(cat $cfg_path)"
 
-  # TODO no coredumps are dropped, but we get a journal entry. Depends
-  # on https://github.com/snapcore/core-base/pull/227 being in the
-  # installed core24 channel already.
+  # Install snap with a service that simply waits
+  core_dump_d=$TESTSLIB/snaps/core-dump-snap
+  BASE=core"$VERSION_ID"
+  sed -i "s/##BASE##/$BASE/" "$core_dump_d"/meta/snap.yaml
```

Comment:
we should revert this change during restore

Reply:
Changed to copying inside the test folder before modifying.